### PR TITLE
New package: mepo-1.3.3.

### PR DESCRIPTION
**Pull request by jason1987d**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-libc
- I built this PR locally for the following non-native architectures: armv7l-libc, armv6l-musl

### CODE REVIEW

File: srcpkgs/mepo/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'mepo'
+pkgname=mepo
+version=1.2.1
+revision=1
+build_style=zig-build
+hostmakedepends="SDL2_image-devel SDL2_gfx-devel SDL2_ttf-devel SDL2-devel libcurl-devel glibc-devel"
+#depends="${hostmakedepends} zig"
+depends="zig"
+short_desc="OSM map viewer for Linux - designed with mobile linux in mind"
```
Comment (classabbyamp) [inline srcpkgs/mepo/template:9]:
```suggestion
short_desc="Fast, simple, and hackable OSM map viewer"
```
from upstream

File: srcpkgs/mepo/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'mepo'
+pkgname=mepo
+version=1.2.1
+revision=1
+build_style=zig-build
+hostmakedepends="SDL2_image-devel SDL2_gfx-devel SDL2_ttf-devel SDL2-devel libcurl-devel glibc-devel"
```
Comment (classabbyamp) [inline srcpkgs/mepo/template:6]: should be makedepends

File: srcpkgs/mepo/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+#depends="${hostmakedepends} zig"
```
Comment (classabbyamp) [inline srcpkgs/mepo/template:7]: rm comment

File: srcpkgs/mepo/template [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'mepo'
+pkgname=mepo
+version=1.3.1
+revision=1
+build_style=zig-build
+makedepends="SDL2_image-devel SDL2_gfx-devel SDL2_ttf-devel SDL2-devel libcurl-devel glibc-devel"
```
Comment (classabbyamp) [inline srcpkgs/mepo/template:6]: why is glibc-devel needed?

File: srcpkgs/mepo/template [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+depends="zig"
```
Comment (classabbyamp) [inline srcpkgs/mepo/template:7]: why is this needed?

**Comment by jason1987d** [comment]
I removed the glibc-devel dependency, and now the two archs failed. They failed to build on my local box as well for these archs. Has this error been encountered before?

**Comment by tranzystorekk** [comment]
Probably worth reporting upstream, as they aren't testing on these platforms

**Comment by jason1987d** [comment]
> Probably worth reporting upstream, as they aren't testing on these platforms

This issue created: https://todo.sr.ht/~mil/mepo-tickets/80
In the meantime, would you recommend excluding those archs with archs= line?

**Comment by classabbyamp** [comment]
no, use a conditional `broken`
